astropy/tests/tests/test_runner.py

```diff
@@ -1,9 +1,11 @@
-from astropy.tests.runner import TestRunner, TestRunnerBase, keyword
+from astropy.tests.runner import TestRunner as T_Runner
```

Comment:
Update for mhvk/astropy#23: CI reports are favorable, see https://travis-ci.org/pllim/astropy/builds/192794283 and https://ci.appveyor.com/project/pllim/astropy/build/1.0.126